engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -251,7 +251,7 @@ <h4 ng-show="repository.content_type !== undefined" translate> Published Reposit
         </select>
       </div>
 
-      <div bst-form-group label="{{ 'SSL Client Key' | translate }}" ng-show="repository.content_type === 'yum'">
+      <div bst-form-group label="{{ 'SSL Client Key' | translate }}" ng-show="repository.content_type === 'yum' || repository.content_type === 'deb'">
```

Comment:
Looking at the backend code, it seems that this is supported on all types, what to just open it up to all?

Reply:
If you think so...